seqr/migrations/0069_remove_sample_dataset_type_and_more.py

```diff
@@ -0,0 +1,83 @@
+# Generated by Django 4.2.13 on 2024-07-12 18:41
+
+from django.conf import settings
+from django.db import migrations, models
+from django.db.models import F
+from django.db.models.functions import Coalesce
+import django.db.models.deletion
+import django.utils.timezone
+
+
+def split_samples(apps, schema_editor):
```

Comment:
Note that this migration works by copying all the non-RNA data into a new table and leaving the RNA data in the old table. This is a bit counterintuitive as, in the end, the table named "sample" (which is the old table name) has all the non-RNA samples.  However, the non-RNA samples have no foreign key references while the RNA samples do, and migrating those was prohibitively slow so it made more sense to leave the RNA samples in their exsting table to preserve their IDs